### PR TITLE
show packages version with new switch: pipsi list --versions

### PR DESCRIPTION
**Pull request by msztolcman**
New switch for `list` command: `--versions`
Added versions to list of packages. Version is determined in time of install, and saved in `venv_path/package_info.json`

It's cause is that i don't know good method to determine package versions - i'm using result of `pip freeze` command, and I don't think calling and parsing it every `pipsi list` call is good enough.

### CODE REVIEW

File: pipsi.py [hunk 1]
```diff
@@ -209,6 +210,34 @@ def link_scripts(self, scripts):
 
         return rv
 
+    def get_package_version(self, venv_path, package):
+        from subprocess import Popen, PIPE
+
+        args = [os.path.join(venv_path, BIN_DIR, 'pip'), 'freeze']
```
Comment (RonnyPfannschmidt) [inline pipsi.py]: at first glance it seems possible that this one might break on windows

**Comment by RonnyPfannschmidt** [comment]
could you please try to accompany this one with a test, at first glance i like what i see,
its nice starting point for many more things to come (yay for per env metadata)

File: pipsi.py (hunk 1 above)
Comment (msztolcman) [inline pipsi.py]: Why? It's used in the same way as in install:

`args = [os.path.join(venv_path, BIN_DIR, 'pip'), 'install']`

I have no easy way to test it on windows, but it should work fine.

Comment (RonnyPfannschmidt) [inline pipsi.py]: indeed, my mistake, sorry for buzzing on that

**Comment by msztolcman** [comment]
Is there some special way to test pipsi? Tests included in `testing` directory fails, and I don't what convention should I follow.

**Comment by msztolcman** [comment]
@RonnyPfannschmidt Is there anything about existing tests, how to run them etc? Or this tests doesn't work and I should create some from the scratch?

**Comment by RonnyPfannschmidt** [comment]
@msztolcman atm i don't recall (i haven't touched the code/tests in a while and there is no travis setup)

**Pull request merged**